lute/book/forms.py

```diff
@@ -83,7 +83,7 @@ def validate_language_id(self, field):  # pylint: disable=unused-argument
             raise ValidationError("Please select a language")
 
     def validate_text(self, field):  # pylint: disable=unused-argument
-        "Throw if missing text and textfile, or if have both."
+        "Throw if textfile is not uploaded and title is empty, or if both are provided."
```

Comment:
This isn't correct, this is validating just text content, which is provided either with the text field or with a text file, not the title.

Reply:
Not sure what I'm missing here :-)